tests/phpunit/unit-tests/api/utilities/beansSanitizePath.php

```diff
@@ -29,21 +29,102 @@ protected function setUp() {
+		);
+		$this->assertSame( $this->prepare_path( __FILE__ ), beans_sanitize_path( __FILE__ ) );
+
+		// test beans' theme root path.
+		$this->assertSame(
+			$this->prepare_path( rtrim( BEANS_ROOT_DIR, DIRECTORY_SEPARATOR ) ),
+			beans_sanitize_path( BEANS_ROOT_DIR )
+		);
+		$directory_separator = '\\' === DIRECTORY_SEPARATOR ? '/' : '';
+		$this->assertSame(
+			$this->prepare_path( BEANS_ROOT_DIR ) . $directory_separator . 'functions.php',
+			beans_sanitize_path( BEANS_ROOT_DIR . 'functions.php' )
+		);
 	}
 
 	/**
-	 * Test beans_sanitize_path() should remove Windows drive.
+	 * Test beans_sanitize_path() should sanitize for filesystem symbolic links.
 	 */
-	public function test_should_remove_windows_drive() {
-		$this->assertSame( '/Program Files/tmp/', beans_sanitize_path( 'C:\Program Files\tmp\\' ) );
-		$this->assertSame( '/Foo/bar/baz/index.txt', beans_sanitize_path( 'D:\Foo\bar\baz\index.txt' ) );
+	public function test_should_sanitize_for_filesystem_symbolic_links() {
+		// test in the ./tests/api directory.
+		$path = __DIR__ . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR;
+		$this->assertSame(
+			$this->prepare_path( BEANS_TESTS_DIR ) . '/unit-tests/api',
+			beans_sanitize_path( $path )
+		);
+
+		// test in the ./tests directory.
+		$path .= '..' . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR;
+		$this->assertSame( $this->prepare_path( BEANS_TESTS_DIR ), beans_sanitize_path( $path ) );
+		$this->assertSame(
+			$this->prepare_path( BEANS_TESTS_DIR ) . '/bootstrap.php',
+			beans_sanitize_path( $path . 'bootstrap.php' )
+		);
+
+		// test in the ./tests/integration-tests directory.
+		$path .= 'integration-tests' . DIRECTORY_SEPARATOR;
+		$this->assertSame(
+			$this->prepare_path( BEANS_TESTS_DIR ) . '/integration-tests',
+			beans_sanitize_path( $path )
+		);
+		$this->assertSame(
+			$this->prepare_path( BEANS_TESTS_DIR ) . '/integration-tests/bootstrap.php',
+			beans_sanitize_path( $path . 'bootstrap.php' )
+		);
+
+		// test in the Beans root directory.
+		$path = BEANS_TESTS_DIR . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR;
+		$this->assertSame(
+			$this->prepare_path( rtrim( BEANS_ROOT_DIR, DIRECTORY_SEPARATOR ) ),
+			beans_sanitize_path( $path )
+		);
+		$directory_separator = '\\' === DIRECTORY_SEPARATOR ? '/' : '';
+		$this->assertSame(
+			$this->prepare_path( BEANS_ROOT_DIR ) . $directory_separator . 'functions.php',
+			beans_sanitize_path( $path . 'functions.php' )
+		);
+	}
+
+	/**
+	 * Prepare the path for different OS environments.
+	 *
+	 * @since 1.5.0
+	 *
+	 * @param string $path Filesystem path to prepare.
+	 *
+	 * @return string
+	 */
+	protected function prepare_path( $path ) {
+
+		// It's not a Windows path. Just return.
+		if ( '\\' !== DIRECTORY_SEPARATOR ) {
+			return $path;
+		}
+
+		$path = wp_normalize_path( realpath( $path ) );
+
+		// Strips off the Windows drive letter.
+		return mb_substr( $path, 2 );
```

Comment:
The `prepare_path()` does not nothing on a Mac or Linux machine.  But on a Windows machine, it has to do the following work for the expected path:

1. Convert it to a real absolute path.
2. Normalize it.
3. Strip of the drive, e.g. `C:`, `D:`, etc.